libs/blocks/marketo/marketo.js

```diff
@@ -123,6 +128,21 @@ export const formSuccess = (formEl, formData) => {
   return false;
 };
 
+export function setProductOfInterest(formData, search = window.location.search) {
+  if (formData[PROGRAM_POI]) return;
+
+  const queryPoi = new URLSearchParams(search).get(QUERY_POI);
+  if (queryPoi === 'clear') {
+    sessionStorage.removeItem(SESSION_POI);
+    return;
+  }
```

Comment:
Clearing and returning without setting a poi seems a little strange. Is this just for dev purposes?

Reply:
Yes, this was for dev purposes, I can remove this.